### ISSUE TITLE
Basic example in docs doesn't work in node 8.9.1-LTS

### ISSUE THREAD

**Issue by professorhantzen** [CityOfZion/neo-js#71]
Hi there,

I believe I'm following the documentation correctly.  I installed neo-blockchain-js using the line:  

`npm install neo-js-blockchain --save`

It installed without any errors, but with some warnings (note, I've redacted the path):

```
npm WARN saveError ENOENT: no such file or directory, open '/Users/[ ... ]/neo/package.json'
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN enoent ENOENT: no such file or directory, open '/Users/[ ... ]/neo/package.json'
npm WARN neo No description
npm WARN neo No repository field.
npm WARN neo No README data
npm WARN neo No license field.

+ neo-js-blockchain@0.0.5
added 37 packages in 2.432s
```

Then I try the following examples from the docs in the node REPL:

```
> var neo = require('neo-js-blockchain');
undefined
> var neoBlockchain = neo('light', 'mainnet');
TypeError: neo is not a function
    at repl:1:21
    at ContextifyScript.Script.runInThisContext (vm.js:50:33)
    at REPLServer.defaultEval (repl.js:240:29)
    at bound (domain.js:301:14)
    at REPLServer.runBound [as eval] (domain.js:314:12)
    at REPLServer.onLine (repl.js:441:10)
    at emitOne (events.js:121:20)
    at REPLServer.emit (events.js:211:7)
    at REPLServer.Interface._onLine (readline.js:282:10)
    at REPLServer.Interface._line (readline.js:631:8)
> neo
{ neo: [Function: neo] }
>
```

What's wrong here?

**Comment by professorhantzen** [comment]
Upon a suggestion in the NEO Slack channel, I've tried:

`var neo = require('neo-js-blockchain').neo;`

Which resulted in:

```
node_modules/neo-js-blockchain/dist/neo.blockchain.node.js:365
         params[scriptHash, params],
               ^

SyntaxError: Unexpected token [
```

And another person suggested:

`var neoBlockchain = new neo('light', 'mainnet')`

Which resulted in a "neo is not a constructor" error.

Why should I need to try different approaches?  What's going wrong with the module as-is?  8.9.1-LTS is the current recommended version of node.js - is the module incompatible?  I also tried in 8.2 but had the same issues.

To be clear, I have a sandboxed environment running with nodenv, so it shouldn't be some kind of module conflict.  I'm essentially npm'ing this module onto a completely fresh installation of node.
